pytorch_lightning/trainer/training_loop.py

```diff
@@ -397,6 +399,7 @@ def train(self):
             # user could press ctrl+c many times... only shutdown once
             if not self.interrupted:
                 self.interrupted = True
```

Comment:
I guess that `self.interrupted` can be removed in the future?

Reply:
ye, more of them could be cleaned...